examples/angular/src/app/app.component.ts

```diff
@@ -4,6 +4,7 @@ import type { WalletSelector, AccountState } from "@near-wallet-selector/core";
 import { setupDefaultWallets } from "@near-wallet-selector/default-wallets";
 import { setupNearWallet } from "@near-wallet-selector/near-wallet";
 import { setupSender } from "@near-wallet-selector/sender";
+import { setupNearFi } from "@near-wallet-selector/nearfi";
```

Comment:
You should add `setupNearFi` to `modules` for it to show.

Reply:
is it a must to support the Angular example too?

Reply:
Yes.